cognite/powerops/custom_modules/power_model_v1/data_models/containers/PriceArea.container.yaml

```diff
@@ -1,8 +1,28 @@
 space: "{{power_ops_models}}"
-externalId: PriceAreaAFRR
-name: PriceAreaAFRR
+externalId: PriceArea
+name: PriceArea
 usedFor: node
 properties:
+  defaultBidConfiguration:
+    type:
+      type: direct
+    nullable: true
+    autoIncrement: false
+    name: defaultBidConfiguration
+  mainPriceScenario:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: mainPriceScenario
+  priceScenarios:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: priceScenarios
   capacityPriceUp:
```

Comment:
Not related to this PR, but capacityPriceUp, capacityPriceDown, activationPriceUp, activationPriceDown, relativeActivation, totalAllocationUp and totalAllocationDown should be per market/product (mFRR, aFRR, FCR-N D-1 and D-2, FCR-D D-1 and D-2), so I think we should remove these from the PriceArea container (and maybe have them in a separate PriceAreaAFRR container for now and add more later) - if they were to live in a broad, general PriceArea container, that would have to be apacityPriceUpAFRR etc.

Reply:
It would also be great if priceScenarios and mainPriceScenario and defaultBidConfiguration got dayAhead as part of their name - in the container, but but in the PriceAreaDayAhead view we could skip the dayAhead part in property names)

Reply:
I think this one is resolved through what we discussed today (one instance/row in the container per market, and assume actual price is part of mainPriceScenario) 👍